### ISSUE TITLE
[BUG] Failing to connect to reddit.com/login

### ISSUE THREAD

**Issue by jacouby** [rdeepak2002/reddit-place-script-2022#368]
All requirements are met
Error is coming from line 492:
```
while True:
                        try:
                            client = requests.Session()
                            client.headers.update(
                                {
                                    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.84 Safari/537.36"
                                }
                            )
                            r = client.get("https://www.reddit.com/login")
                            login_get_soup = BeautifulSoup(r.content, "html.parser")
                            csrf_token = login_get_soup.find(
                                "input", {"name": "csrf_token"}
                            )["value"]
                            data = {
                                "username": username,
                                "password": password,
                                "dest": "https://new.reddit.com/",
                                "csrf_token": csrf_token,
                            }

                            r = client.post(
                                "https://www.reddit.com/login",
                                data=data,
                                proxies=proxy.get_random_proxy(),
                            )
                            break
                        except Exception:
                            logger.error(
                                "Failed to connect to websocket, trying again in 30 seconds..."
                            )
                            time.sleep(30)
```

Log:
`2022-04-03 21:44:33.863 | ERROR    | __main__:task:492 - Failed to connect to websocket, trying again in 30 seconds...
2022-04-03 21:44:35.856 | ERROR    | __main__:task:492 - Failed to connect to websocket, trying again in 30 seconds...
2022-04-03 21:45:02.077 | ERROR    | __main__:task:492 - Failed to connect to websocket, trying again in 30 seconds...
2022-04-03 21:45:03.981 | ERROR    | __main__:task:492 - Failed to connect to websocket, trying again in 30 seconds...
2022-04-03 21:45:05.965 | ERROR    | __main__:task:492 - Failed to connect to websocket, trying again in 30 seconds...
2022-04-03 21:45:32.642 | ERROR    | __main__:task:492 - Failed to connect to websocket, trying again in 30 seconds...
2022-04-03 21:45:34.089 | ERROR    | __main__:task:492 - Failed to connect to websocket, trying again in 30 seconds...
2022-04-03 21:45:36.135 | ERROR    | __main__:task:492 - Failed to connect to websocket, trying again in 30 seconds...
2022-04-03 21:46:02.783 | ERROR    | __main__:task:492 - Failed to connect to websocket, trying again in 30 seconds...`

**Comment by colingoodman** [comment]
Same issue as the now closed #291. I am also having this issue on the latest pull.

**Comment by TheForgottenKing** [comment]
I have the same issue on the latest pull.

**Comment by NotADinosaur** [comment]
same issue :(

**Comment by histefanhere** [comment]
Same issue here from main branch

**Comment by Risewill23** [comment]
This will get fixed in #375
